packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswerRephraseButtons/quanticGeneratedAnswerRephraseButtons.js

```diff
@@ -8,13 +8,13 @@ import {LightningElement, api} from 'lwc';
 
 /**
  * The `QuanticGeneratedAnswerRephraseButtons` component displays a set of rephrase buttons allowing the modification of the answer style of the generated answer.
- * @category Internal
+ * @category Search
```

Comment:
Same thing for this component, it's more a component to reduce the complexity of the Quantic Generated Answer and it's not meant to be directly handled by the end users.